.github/workflows/changelog_test.yml

```diff
@@ -0,0 +1,38 @@
+name: Changelog check
+
+on:
+  # allows us to run workflows manually
+  workflow_dispatch:
+  pull_request:
```

Comment:
So this will only run when a pull request is created. On subsequent pushes to that PR it will not run. Not sure if this is the behavior you want or not.

Reply:
May want to add a `push` trigger